### PR TITLE
Issue/250 - Change file path for interactive thumbnails

### PR DESCRIPTION
**Pull request by hayleyavw**
This PR also tidied up the implementation of the interactive processor to make better use of the `GenericTagBlockProcessor` (following the same method as the recently updated image and panel processors), and also added many more tests as this interactive had arguably insufficient testing (likely because until we were unsure how it was going to be used!)

The path to interactive thumbnails is now either `thumbnail.png` by default or manually specified by overriding the the `interactive-thumbnail-path.html` file (similar to how an image path can be overwritten using the `relative-file-link.html` template file).

### CODE REVIEW

File: verto/tests/InteractiveTest.py [hunk 1]
```diff
@@ -23,9 +25,192 @@ def __init__(self, *args, **kwargs):
...
+                "binary-cards"
+            },
+            'images': {
+                'binarycards.png'
+            },
+            'page_scripts': set(),
+            'scratch_images': set()
+        }
+        self.assertEqual(self.verto_extension.required_files, required_files)
+        '''
```
Comment (JackMorganNZ) [inline verto/tests/InteractiveTest.py]: This test has been commented out.

**Comment by hayleyavw** [comment]
It was entirely to keep things broken down into separate components, and also so that the user can write a custom thumbnail path template file, depending on their own file structure 

The file path for the thumbnail has not been confirmed even in our own django system, so I couldn't commit to defining the file path in the code, and didn't want the user to have to define it manually for every interactive as it could change, this way there is only one place where it will have to change.

**Comment by JackMorganNZ** [comment]
First part makes sense (don't understand the second bit) I'll clarify some points now the new image PR has been created.

The previous relative file template was separated as it was used my multiple templates, however the new interactive relative file template is just used by one. Also the new PR for image template changes has the relative file logic within each template, which on first glance looks repetitive, however in our own CS Unplugged repo I plan to use different relative link logic depending if it's an inline image or not.

For consistency with the image templates, the logic could be within the parent template as it's only used there. However if you wish to keep them separate, this is fine too.

Happy with your judgement on this, let me know which way suits you.

**Comment by hayleyavw** [comment]
I'll just change it to match how the new image processor works, for consistency.

**Pull request merged**